notebooks/latent-consistency-models-image-generation/lcm-lora-controlnet.ipynb

```diff
@@ -1228,16 +1228,16 @@
     }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    to_quantize = quantization_widget(skip_for_device)
</pre><p>not</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>